### PR TITLE
Tp 1952 tp 1953 delegate vote

### PR DESCRIPTION
**Pull request by ftreguer**

### CODE REVIEW

**Comment by dlecan** [comment]
https://spacelephant.tpondemand.com/entity/1952

File: src/commands/delegate/unvote.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { AbstractDelegateVoteCreateCommand } from "../../abstract-vote";
+import { BaseCommand } from "../../baseCommand";
+import { getNetworksListListForDescription } from "../../utils";
+
+export class DelegateUnvoteCreateCommand extends AbstractDelegateVoteCreateCommand {
+    public static description = "Vote for a delegate with his Unikname or unikid";
```
Comment (peterjah) [inline src/commands/delegate/unvote.ts]: description à changer

**Review by peterjah — APPROVED** [review]

**Review by peterjah — APPROVED** [review]

**Review by peterjah — APPROVED** [review]

**Review by peterjah — APPROVED** [review]

**Review by dlecan — APPROVED** [review]

**Review by dlecan — DISMISSED** [review]
To test

**Comment by ftreguer** [comment]
> To test

?

**Review by peterjah — APPROVED** [review]

**Pull request merged**